scraping/spiders/france_red_cross.py

```diff
@@ -56,31 +57,31 @@ class FranceRedCrossSpider(scrapy.Spider):
     def parse(self, response, **kwargs):
         yield from parse_points(response)
 
-        next_page = response.css('li.next a::attr(href)').get()
+        next_page = response.css("li.next a::attr(href)").get()
         if next_page:
             yield response.follow(next_page, callback=self.parse)
 
 
 def parse_points(response):
-    blocks = response.css('article.item')
+    blocks = response.css("article.item")
     for block in blocks:
         yield parse_point(block)
 
 
 def parse_point(block):
-    name = block.css('h2 a span::text').get().strip().title()
+    name = block.css("h2 a span::text").get().strip().title()
 
     point = {
-        'name': name,
-        'country': 'France',
-        'organizations': ['Red Cross France'],
-        'lat': '',
-        'lng': '',
+        "name": name,
+        "country": "France",
+        "organizations": ["Red Cross France"],
```

Comment:
Every single point there is actually operated by Red Cross?

Reply:
Good question, I had an impression that most of them are at least closely affiliated (having same TLD websites, etc), but honestly I don't really know…

Reply:
Yes, it's confirmed, they are all red cross